src/tir/transforms/lower_sparse_tir.cc

```diff
@@ -224,8 +285,10 @@ class IndexTransformer : public StmtExprMutator {
   PrimExpr AccumulateLowerIndex(PrimExpr prev_lowered_index, const SparseBuffer& sp_buffer, int dim,
                                 PrimExpr index) {
     const Axis& axis = sp_buffer->axes[dim];
-    if (axis->IsInstance<DenseFixedAxisNode>() || axis->IsInstance<SparseFixedAxisNode>()) {
+    if (axis->IsInstance<DenseFixedAxisNode>()) {
       return ana_.Simplify(std::move(prev_lowered_index) * axis->length + std::move(index));
+    } else if (const auto* sf_axis = axis.as<SparseFixedAxisNode>()) {
+      return ana_.Simplify(std::move(prev_lowered_index) * sf_axis->num_cols + std::move(index));
```

Comment:
btw, I suppose renaming this `num_cols` to `nnz_cols`(number of nonzero columns) to avoid ambiguity.

Reply:
Okay. Will change later.